Janmaijai rana/Intro.md

```diff
@@ -0,0 +1,17 @@
+
+
+
+
+## INTRODUCTION ##
+ 
+   * Name : Janmaijai Rana
+   * I am currently doing my B.TECH (Computer Science) from **CENTRAL UNIVERSITY OF HARYANA.** 
+   * My hobbies are Listening Music, Machine Learning, Deep Learning. I love to make applications that can solve real world                 problems.
```

Comment:
This is exactly the same as my intro?
Reduce the gap between world and problems